### PR TITLE
Test Suite

### PR DESCRIPTION
**Pull request by mpstanford**

### CODE REVIEW

**Comment by mpstanford** [comment]
Forgot to tag you guys @mbbph @dbooyah93  @vargas055

File: test/client.test.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    const myImage = shallow(<Image productId={0} />);
+    const instance = myImage.instance();
+    expect(instance).toBeInstanceOf(Image);
+  });
+
+  test('checks if an image is rendered', () => {
+    const myImage = shallow(<Image productId={0} />);
+    const instance = myImage.instance();
+    expect(instance).toBeInstanceOf(Image);
+  });
```
Comment (mbbph) [inline test/client.test.jsx:32]: these two seem to be duplicates

**Review by mbbph — APPROVED** [review]
Test suite looks great, seems very comprehensive. Two of the client-side tests seem to be identical, but that's not a big deal.

File: test/client.test.jsx (hunk 1 above)
Comment (mpstanford) [reply test/client.test.jsx:32]: oh you're right! good catch. Thank you! Will delete one of them and then merge.

**Pull request merged**